diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/InlineClassesRule.kt

```diff
@@ -37,7 +38,8 @@ class InlineClassesRule(private val configRule: List<RulesConfig>) : Rule("inlin
         emitWarn = emit
         isFixMode = autoCorrect
 
-        if (node.elementType == CLASS) {
+        val configuration by configRule.getCommonConfiguration()
+        if (node.elementType == CLASS && configuration.kotlinVersion.isAtLeast(ktVersion.major, ktVersion.minor, ktVersion.patch)) {
```

Comment:
Since KotlinVersion [implements `Comaparable`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-kotlin-version), can we compare `configuration.kotlinVersion` with `ktVersion` directly?

Reply:
Yes, fixed